### DISCUSSION TITLE
idea to send email from from case

### DISCUSSION THREAD

**Discussion by xSunShine** [Security-Onion-Solutions/securityonion#9810] [Ideas]
Hello,

It would be good if we could send email a case after we collect everything inside case to have an option to send that case via email or any other platfrom to share with others.

**Comment by InfosecGoon** [comment]
All of the contents of a case are stored in Elasticsearch, so you should be able to pull it all out via Kibana or some other reporting tool if you like. Is the use case here to share it with another analyst, or to send it into another system for further processing?

**Reply by xSunShine** [reply]
Yes i known i can pull all from elastic, but my idea is as example for a soc or cert that all gather information from case can be sent from case via email or someother platform
